spec/cypress/integration/admin/LeadProviderUserManagement.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Admin user modifying lead provider users
+  Admin users should be able to create and delete lead provider users
+
+  Background:
+    Given I am logged in as an "admin"
+    And scenario "admin/suppliers" has been run
+    And I am on "lead provider users index" page
+
+  Scenario: Creating a new lead provider user
+    When I click on "create supplier user button"
+    Then I should be on "new lead provider user" page
+    And the page should be accessible
+
+    When I type "Lead" into "supplier name input"
+    And I click on "autocomplete dropdown item" containing "Lead Provider 1"
+    And I click the submit button
+    Then I should be on "new lead provider user details" page
+    And the page should be accessible
+
+    When I type "John Smith" into "name input"
+    And I type "j.s@example.com" into "email input"
+    And I click the submit button
+    Then I should be on "new lead provider user review" page
+    And the page should be accessible
+    And "page body" should contain "John Smith"
```

Comment:
"page body" => "the page"?

Reply:
"page body" was requested by @kicferk1 in another PR 😆